store-implementation/map-store/src/test/java/uk/gov/gchq/gaffer/mapstore/MapStoreTest.java

```diff
@@ -44,7 +45,7 @@ public void testTraits() throws StoreException {
                 StoreTrait.TRANSFORMATION,
                 StoreTrait.POST_TRANSFORMATION_FILTERING,
                 StoreTrait.MATCHED_VERTEX));
-        assertEquals(expectedTraits, mapStore.getTraits());
+        assertEquals(expectedTraits, mapStore.TRAITS);
```

Comment:
I think maybe this can stay as `getTraits`? Or should at least be `MapStore.TRAITS`

Reply:
Yes this needn't be changed. It's left over from a previous commit that I otherwise undid.